docs/source/reference/python.rst

```diff
@@ -9,6 +9,8 @@ This list is incomplete. You can help by *expanding it*!
 Atoms and cells
 ===============
 
+.. autofunction:: rascal.neighbourlist.structure_manager.AtomsList
```

Comment:
I think this should be `.. autoclass:: rascal.neighbourlist.structure_manager.AtomsList`